src/views/notifications/Histories.vue

```diff
@@ -0,0 +1,254 @@
+                            <img
+                              alt=""
+                              class="rounded-full"
+                              :src="this.history_notification_user?.profile_picture ?? require('@/assets/images/avatar.png')"
+                            />
+                          </div>
+                          <div class="ml-5">
+                            <div class="w-24 sm:w-40 truncate sm:whitespace-normal font-medium text-lg">
+                              {{ this.history_notification_user?.name }}
+                            </div>
+                            <div class="text-gray-600">{{ this.history_notification_user.role?.name }}</div>
+                          </div>
+                        </div>
+                        <div class="mt-6 lg:mt-0 flex-1 dark:text-gray-300 px-5 border-l border-r border-gray-200 dark:border-dark-5 border-t lg:border-t-0 pt-5 lg:pt-0">
+                          <div class="font-medium text-center lg:text-left lg:mt-3">
+                            Contact Details
+                          </div>
+                          <div class="flex flex-col justify-center items-center lg:items-start mt-4">
+                            <div class="truncate sm:whitespace-normal flex items-center">
+                              <MailIcon class="w-4 h-4 mr-2"/>{{ this.history_notification_user?.email }}
+                            </div>
+                            <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                              <UserIcon class="w-4 h-4 mr-2"/>{{ this.history_notification_user?.pre_name }} {{ this.history_notification_user?.last_name }}
+                            </div>
+                            <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                              <HashIcon class="w-4 h-4 mr-2"/>User ID: {{ this.history_notification_user?.id }}
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <div class="grid grid-cols-12 gap-6 p-5 flex">
+                    <!-- BEGIN: Users Layout -->
+                    <div
+                      v-for="notification in notifications"
+                      v-bind:key="notification.id"
+                      class="intro-y col-span-12 md:col-span-6 lg:col-span-4"
+                    >
+                      <div class="box">
+                        <div class="flex items-start pt-5 pl-1">
+                          <div class="w-full flex flex-col lg:flex-row items-center">
+                            <div class="lg:ml-4 text-center lg:text-left mt-3 lg:mt-0">
+                              <a href="" class="font-medium">{{ notification.title.substring(0,100) }}{{ notification.title > 100 ? '...' : '' }}</a>
```

Comment:
You might use css internals function for that instead of using javascript for this task: `text-overflow: ellipsis;`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/7ce1e76f3284139350f2ec7415415ff6b695b571